container.go

```diff
@@ -244,18 +244,13 @@ func fetchContainer(pod *Pod, containerID string) (*Container, error) {
 		return nil, errNeedContainerID
 	}
 
-	fs := filesystem{}
-	config, err := fs.fetchContainerConfig(pod.id, containerID)
-	if err != nil {
-		return nil, err
-	}
-
-	container, err := createContainer(pod, config)
-	if err != nil {
-		return nil, fmt.Errorf("failed to create container with config %v in pod %v: %v", config, pod.id, err)
+	for _, c := range pod.containers {
+		if containerID == c.id {
+			return c, nil
+		}
 	}
 
-	return container, nil
+	return nil, fmt.Errorf("Could not find the container in the pod's containers list")
```

Comment:
Could you add `containerID` to the returned error (and maybe `pod.ID()`)?

Reply:
Oh yeah sorry I forgot that ! Done !